### DISCUSSION TITLE
Movement Blocker

### DISCUSSION THREAD

**Discussion by goldjunge91** [5axes/CustomSupportCylinder#24] [Ideas]
Hi,
i nice work really helpfull,
i'm thinking if its Possible to create a movement blocker where the nozzle have to travel around ?

**Comment by 5axes** [comment]
Yes it's possible , with the rights parameters and if you use a Combing Mode.
![image](https://user-images.githubusercontent.com/11015345/193050383-a02aba8b-0937-4a69-81de-a9990d3004d1.png)

But in this case It Should be I thing A new Plugin

**Comment by goldjunge91** [comment]
uhhh nice, yes and now think about a fan duct where you don't wan't stringing inside 👯‍♂️ 
![Screenshot 2022-09-29 164243](https://user-images.githubusercontent.com/30201929/193064099-5c2466f0-1466-4c29-bc1c-457205701271.png)

**Reply by 5axes** [reply]
Sometimes things are not possible 🙄

**Comment by gr5** [comment]
It's possible - just do the exact same thing as shown in the picture at top but use a much more complicated mesh that fits inside the area where you want no strings.  Also make sure "drop models to build plate" is turned off.
